python/ray/serve/storage/tests/test_kv_store.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+import pytest
+from ray.serve.storage.kv_store import (RayInternalKVStore, RayLocalKVStore,
+                                        RayS3KVStore)
+from ray.serve.tests.conftest import (  # noqa: F401 imported but unused
+    serve_instance, shared_serve_instance)
```

Comment:
why do we import `shared_serve_instance`? I don't think it's used.

Reply:
test will failing complaining `shared_serve_instance` not imported if it's not there

Reply:
huh so weird. pytest works in mysterious ways

Reply:
Hmmm they should not be imported! If it fails with import error make sure serve instance is an argument name for the test function first. So that the serve_instance variable comes from function argument not directly import.